README.md

```diff
@@ -313,6 +313,19 @@ This is a costly check and requires an additional pass through the table.
 |-----|------|-------------|
 | `columns` | Array[String] | Each set of values in these columns must be unique.
 
+#### `sumOfNumericColumnCheck`
+
+This check is used to ensure that the _sum_ of a numeric column is over or under a threshold or that it is
+within or outside a range.
+
+| Arg             | Type   | Description                                            |
+|-----------------|--------|--------------------------------------------------------|
+| `column`        | String | The column to consider                                 |
+| `thresholdType` | String | One of: `over`, `under`, `between`, `outside`          |
+| `threshold`     | Number | Single high water mark required for `over` and `under` |
+| `lowerBound`    | Number | Lower bound required for `between` and `outside`       |
+| `upperBound`    | Number | Upper bound required for `between` and `outside`       |
```

Comment:
This seems inconsistent with rangeCheck.  Can the lowerBound and upperBound fields not just be optional so that the thresholdType is inferred from which bounds are supplied?  Also, this doesn't have the `inclusive` boolean that rangeCheck has.

Reply:
I think ideally this check would be completely identical to rangeCheck but with an accumulator.  Are the differences important?

Reply:
My goal was to keep the configuration simple and explicit:

```yaml
type: sumOfNumericColumnCheck
column: usage
thresholdType: over
threshold: 2345252234
```
or
```yaml
type: sumOfNumericColumnCheck
column: usage
thresholdType: between
lowerBound: 2345252234
upperBound: 76545678987
```

I can add an `inclusive` flag. My original attempt at this was inclusive explicitly but this solution isn't.

RangeCheck is row-based and SumOfNumericColumnCheck is column-based. They are different archetypes: the row-based checks get passed every row in the original data frame while the column-based checks get passed the output of an aggregate query. The `quickCheck` method signature alludes that they're the same but they're really not.

Reply:
35661cf adds it.